### DISCUSSION TITLE
New updated image in the bucket is not showing in client side unless i open an icognito window

### DISCUSSION THREAD

**Discussion by ricardosikic** [supabase/supabase#14099] [Questions]
Hello, I'm having some issues when i update an image at front. When i update an image using the updating function from supabase it updates the image data properly keeping the file name (the same file name stored into the db) but in the client side the change made into the bucket does not comes up unless i open a new incognito window. The function that i'm using for update the bucket was taken from supabase docs, the only way to see the image updated in client side is disabling cache in dev tools in the browser. I'm using:

```
const { data, error } = await supabase
  .storage
  .from('avatars')
  .update('public/avatar1.png', avatarFile, {
    cacheControl: '3600',
    upsert: false
  })
```

<img width="559" alt="Screen Shot 2023-05-02 at 23 21 10" src="https://user-images.githubusercontent.com/36648562/235826965-a776c9be-fe29-4e60-8f9e-9fd64da60177.png">

**Comment by GaryAustin1** [comment]
Your image is still cached in the CDN/browser.  
https://supabase.com/docs/guides/storage/cdn#cache-duration
Shows some cache "busting" ideas if you need the new image immediately.

**Reply by ricardosikic** [reply]
Hi! thanks again @GaryAustin1 for answer 💪  i read the docs. I did a work around, perhaps could be a not elegant solution to update the image, but allowed me to change the filename, file data obviusly and fixed the delay when i "update" the image for another one. And the solution was 🥁 🥁 :
1. I have called the delete function to delete the image that i wanted to "update" from the bucket
2. Then I have called the upload function for the new image
